homeassistant/components/system_bridge/sensor.py

```diff
@@ -122,6 +136,15 @@ def memory_used(data: SystemBridgeCoordinatorData) -> float | None:
     return None
 
 
+def get_attr_if_available(
+    data: SystemBridgeCoordinatorData, key: str
+) -> str | float | None:
+    """Return the attribute if available."""
+    if hasattr(data, key):
+        return getattr(data, key)
+    return None
```

Comment:
Use the third parameter of `getattr` to set a default value.
This method is not needed and can be removed.